moon/bulk_test.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+# NASA qualification round number
+ROUND=3
+
+# after how long should the simulation give up (in seconds); note the round is allowed to run for up to 45 minutes
+TIMEOUT=1800
```

Comment:
why do you have such low timeout? Is it simulated time?

Reply:
It's 30 minutes. For Round 3 I think it's not worth waiting any longer because we should aim to solve it faster than in 30 minutes. This long a time means that it drove into some crater it cannot get out of or some other problem that we should be solving, not that it truly took more than 30 minutes or efficient work. 

Of course, for Rounds 1 and 2 we will be running to the last moment and there changing it to 45 mins makes sense.